spec/factories/access_requests.rb

```diff
@@ -0,0 +1,20 @@
+FactoryBot.define do
```

Comment:
Filename should match the factory name i.e. singular `access_request .rb`

Reply:
Should they? I've always created factories with plural names, if nothing else it helped with finding the files using fuzzy search in my editor. This is also the way we do them on mc-be. Considering a file here may contain multiple factories, maybe it's ok for them to be plural?

Reply:
Having given it a minutes thought, even though I'd probably lean towards pluralisation, I think what's most important is keeping them standard. Happy to leave this up to Tim or a group consensus.

Reply:
can of worms?

singular vs pain, i opt for consistence 

![image](https://user-images.githubusercontent.com/470137/57923987-b1d06780-789b-11e9-8e43-a5a862434086.png)

Reply:
@misaka consistently wrong is better than inconsistent. Right now we have a mix as @defong points out above.

Reply:
I wasn't arguing against consistency, I was pointing out that we generally use the plural form, in mc-be and mc-fe, so I think we should do that consistently. :) Sorry for my use of "standard" there instead of "consistent".

Reply:
We're on the same page so its all good 💪

Reply:
dang. we copied what is in backend https://github.com/DFE-Digital/manage-courses-backend/tree/master/spec/factories